README.md

```diff
@@ -1340,6 +1340,7 @@ Update Time, five active automations, webhooks.
   * [CockroachDB Cloud](https://www.cockroachlabs.com/pricing/) — Free tier offers 50 million RUs and 10 GiB of storage (same as 15$ worth) free per month. ([What's the Request Units](https://www.cockroachlabs.com/docs/cockroachcloud/metrics-request-units.html))
   * [Neo4j Aura](https://neo4j.com/cloud/aura/) — Managed native Graph DBMS / analytics platform with a Cypher query language and a REST API. Limits on graph size (50k nodes, 175k relationships).
   * [Neon](https://neon.tech/) — Managed PostgreSQL, 0.5 GB of storage (total), 1 Project ,10 branches, Unlimited Databases, always-available primary branch ( Auto suspend after 5 minutes), 20 hours of Active time per month (total) for non-primary branch compute.
+  * [Prisma Postgres](https://prisma.io/postgres) - Super fast hosted Postgres built on unikernels and running on bare metal, 1GB storage, 10 databases, integrated with Prisma ORM.
```

Comment:
```
First 100k operations* free
then $18 / million operations
...
*An operation is each time you interact with your database.
```

This seems quite low to me

Reply:
@ripienaar An operation is not the same as a query. On average, 10 db queries = 1 ORM operation. You can read more here: https://www.prisma.io/blog/operations-based-billing

Additionally, in some areas, the free tier is larger than what others offer. For example, storage is 2x vs Neon/Supabase. Ingress/Egress is included, we don't charge for scaling your CPU needs up and down, etc, etc.

Prisma Postgres is already integrated into marketplaces like Vercel marketplace, and there's widespread adoption. The free tier is robust enough for the criteria that you have listed here.